### PR TITLE
Add themoviedb.org as source for data + images

### PR DESCRIPTION
**Pull request by kmacinnis**
Preferences for series plot, and episode plots, and ratings,
Will automatically use tmdbs as fallback for a few other things
Will download episode images from both tvdb and tmdb.

Closes https://github.com/plexinc-agents/TheTVDB.bundle/issues/15

### CODE REVIEW

File: Contents/Code/__init__.py [hunk 1]
```diff
@@ -25,13 +26,37 @@
...
+TMDB_TV_EPISODE_URL = ('{base}/tv/{{tmdb_id}}'
+    '/season/{{season}}/episode/{{episode}}?'
+    'api_key={api_key}').format(base=TMDB_BASE_URL, api_key=TMDB_API_KEY)
+TMDB_TV_EPISODE_IMAGES_URL = ('{base}/tv/{{tmdb_id}}'
+    '/season/{{season}}/episode/{{episode}}/images?'
+    'api_key={api_key}').format(base=TMDB_BASE_URL, api_key=TMDB_API_KEY)
+
+
+
+# Movies -- Here for remindery purposes only.  Remove when OTA-ishness achieved.
```
Comment (maxGreenblatt) [inline Contents/Code/__init__.py]: I assume these can be removed now?

File: Contents/Code/__init__.py [hunk 2]
```diff
@@ -519,47 +564,74 @@ def ParseSeries(self, media, series_data, lang, results, score):
...
+      tmdb_runtime = None
+    metadata.duration = tvdb_runtime or tmdb_runtime
 
-    # TODO Couldn't find this in the new API - @maxg 2015-10-14
-    try: metadata.rating = float(series_data['rating'])
-    except: pass
+    metadata.rating = get_rating(tvdb_series_data, tmdb_series_data)
 
-    metadata.genres = series_data.get('genre', [])
+    metadata.genres = tvdb_series_data['genre']
```
Comment (maxGreenblatt) [inline Contents/Code/__init__.py]: I _think_ the framework expects this to be a `[]`, so you might want to swap this back to `metadata.genres = tvdb_series_data.get('genre', [])`

File: Contents/DefaultPrefs.json [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    "label": "Series Plot Summary Source: ",
+    "type": "enum",
+    "values": [
+      "",
+      "The TV Database",
+      "The Movie Database"
+    ],
+    "default": "The TV Database"
+  },
+  {
```
Comment (maxGreenblatt) [inline Contents/DefaultPrefs.json]: no metrics in this bundle, so you don't need this pref

**Comment by maxGreenblatt** [comment]
Some minor fixes, but otherwise :eggplant: and rebase (try to make sure this is a ff-merge after you rebase)

File: Contents/Code/__init__.py (hunk 2 above)
Comment (kmacinnis) [inline Contents/Code/__init__.py]: Good point.  I missed that.

**Pull request merged**